libs/src/sdk/sdk.ts

```diff
@@ -126,15 +127,29 @@ const initializeServices = (config: SdkConfig) => {
       ? new AppAuth(config.apiKey, config.apiSecret)
       : new Auth()
 
-  const storageNodeSelector = new StorageNodeSelector({
-    auth: config.services?.auth ?? defaultAuthService
+  const defaultDiscoveryNodeSelector = new DiscoveryNodeSelector()
+
+  const defaultStorageNodeSelector = new StorageNodeSelector({
+    auth: config.services?.auth ?? defaultAuthService,
+    discoveryNodeSelector:
+      config.services?.discoveryNodeSelector ?? defaultDiscoveryNodeSelector
+  })
+
+  const defaultEntityManager = new EntityManager({
+    ...defaultEntityManagerConfig,
+    discoveryNodeSelector:
+      config.services?.discoveryNodeSelector ?? defaultDiscoveryNodeSelector
```

Comment:
`defaultEntityManagerConfig` has `discoveryNodeSelector` right? should we remove it from there if we're always going to set it here?

Reply:
We need it there to satisfy the type, otherwise would have to make `discoveryNodeSelector` optional on `EntityManagerConfig`